admin-ui/plugins/admin/redux/audit/Resources.tsx

```diff
@@ -1,4 +1,6 @@
 export const SCRIPT = 'custom-script'
 export const API_ROLE = 'api-role'
+export const API_CONFIG = 'api-config'
 export const API_PERMISSION = 'api-permission'
 export const API_MAPPING = 'api-role-mapping'
+export const ADMIN_UI_CONFIG = 'admin_ui_config'
```

Comment:
Can we use a name related to cedarling like ADMIN_UI_CEDARLING_CONFIG = 'admin_ui_cedarling_config'